testkit/src/main/scala/org/bitcoins/testkit/rpc/BitcoindRpcTestUtil.scala

```diff
@@ -103,6 +105,7 @@ trait BitcoindRpcTestUtil extends BitcoinSLogger {
       if (blockFilterIndex)
         conf + """
                  |blockfilterindex=1
+                 |peerblockfilters=1
```

Comment:
what does this setting do?

Reply:
This makes it so we can send block filters over the network